database/database.py

```diff
@@ -0,0 +1,56 @@
+import pyodbc
+import os
+import pandas as pd
+
+server_name = 'datascienceforschool.cozlulgmmrar.us-east-2.rds.amazonaws.com'
+user_name = 'boys'
+password = 'rule'
+
+# Inserts into database
+def insert_database(query):
+
+    # Connect to database
+    conn = connect_database()
+
+    # Create database cursor
+    cursor = conn.cursor()
+
+    # Execute insertion and commit to database
+    cursor.execute(query)
+    conn.commit()
+
+    # Close connection
+    disconnect_database(conn)
```

Comment:
Rather than connecting and disconnecting/closing the connection each time you need it, it's probably easier to just create a connection object in the main script and pass it in as a param.

It's a very minor thing though, so I wouldn't worry about it.